aerogear/mobile-developer-console
# Added ability to dynamically add favicons.

## Motivation
Added ability to dynamically change the favicon.  Renamed the Branding logo to be consistent with the other logo names. 

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Verified that the favicon changes based on configuration settings.


## Checklist:

- [ X ] Code has been tested locally by PR requester
